frontend/public/annotator/index.html

```diff
@@ -244,11 +276,9 @@ <h3 class="text-base font-semibold">How do you want this to change?</h3>
 				selectedElements.push(element)
 				clearInspector()
 				setTimeout(() => {
-					if (inspectorEnabled) {
-						window.theRoom.start()
-					}
```

Comment:
I'm forgetting why I had this...  it was probably for adding multiple comments at one point before submitting.  This is cleaner.

Reply:
https://github.com/wandb/openui/blob/3fc829be3e30aa8caf8aaa902136cc333c75ea21/frontend/src/components/HtmlAnnotator.tsx#L206-L210
line 209 set inspectorEnabled to false ,so I deleted this